code/WorkInProgress/infernal_contracts.dm

```diff
@@ -165,7 +165,7 @@ proc/is_weak_rollable_contract(type)
 	if(C.mind)
 		shake_camera(C, 20, 16)
 		boutput(C, "<font color=red>[screamstring]</font>")
-		boutput(C, "<i><b><font face = Tempus Sans ITC>You have sold your soul and become an avatar of evil! Spread darkness across the land!</font></b></i>")
+		boutput(C, "<span style=\"color:purple; font-size:150%\"><i><b><font face = Tempus Sans ITC>You have sold your soul and become an antagonist and an avatar of evil! Spread darkness across the land!</font></b></i></span>")
```

Comment:
umm, this is a big change! they weren't antagonists before!

Reply:
Gerhazo said they were when I asked and the message about not being an antagonist never actually reached the user thanks to improper sequencing.

We decided to resolve the ambiguity, so I clarified the original intention.

This is also why I left the comment down below to record the change in-case there was internal disagreement in the admin team

Reply:
ahh ok i was confused on the comment even after you explained it the first time, this makes more sense now, ty